introspection-engine/connectors/sql-introspection-connector/src/introspection/models.rs

```diff
@@ -1,7 +1,9 @@
-use super::scalar_field;
```

Comment:
Sorry for the messy diff. Moved the single model rendering to a separate function.